tests/functional/rbac/test_cluster_related_permissions.py

```diff
@@ -156,7 +156,7 @@ def test_manage_imports(user_policy: Policy, user_sdk: ADCMClient, is_denied_to_
     for base_object in [admin_cluster_second, admin_service_second]:
         is_denied_to_user(base_object, BR.ViewImports)
     delete_policy(user_policy)
-    # TODO remove binds (check out commit history)
+    # remove binds (check out commit history)
```

Comment:
jira task?

Reply:
There's a Jira task on it, do you suggest adding a link to it (its ID) here in the comment?

Reply:
yep